bin/kano-post-clean

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+# kano-post-clean
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Creates a clean post install environment
+# by enabling autocleaning
+
+
+# Clean the retrieved .deb packages from the local repo
+# and delete all .deb files from /var/cache/apt/archives
+# to free up important space
+post_clean() {
+
+    # Ensure there's no lock enabled
+    sudo rm /var/cache/apt/archives/lock &&
+
+    sudo apt-get clean &&
+    sudo apt-get autoclean;
```

Comment:
So these are not actually triggered inside the updater? Perhaps in the `commands/install.py`?

Reply:
nop, they are executed through the `APT-conf` instructions that the updater can run before/after `apt` runs `dpkg`. I believe it's more standard this way than add another cryptic function inside `install.py`.